apis/course/README.md

```diff
@@ -0,0 +1,576 @@
+# Feature: Course API
+
+## Description:
+
+Define an API for Signal K client applications to be able to set a destination or navigate a route that provides the following methods:
+
+- Set a position (lat, lon) as a destination
+- Reference a waypoint from `/resources/waypoints` as a destination
+- Activate a route by supplying a reference from `/resources/routes`
+- Specify to follow route points in reverse order
+- Select the point within the route to be the current destination
+- Increment / decrement the point in the route to be the current destination.
+- Clear / cancel the course
+- Query the current course details.
+
```

Comment:
Do we want to add arrivalCircle as an independent setting, like a default arrivalCircle, if none is supplied when setting destination?

Reply:
If this API is embedded in the server then a group of course related settings in the admin UI for would be good to provide defaults so the settings would be read at startup.